source/manual/fall-back-to-aws-cloudfront.html.md

```diff
@@ -1,25 +0,0 @@
----
```

Comment:
Could you put a redirect in `config/tech-docs.yml`, just in case there are any incoming links to this page from random google docs and such?